shared/common-adapters/usernames.js

```diff
@@ -205,14 +204,12 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
 
 const styles = styleSheetCreate({
   andStyle: platformStyles({
-    common: {
-      marginLeft: globalMargins.xtiny,
-      marginRight: globalMargins.xtiny,
+    isElectron: {
+      textDecoration: 'none',
+      fontWeight: 'inherit',
     },
-    isElectron: {textDecoration: 'none'},
   }),
   commaStyle: platformStyles({
-    common: {marginRight: 1},
     isElectron: {textDecoration: 'none'},
```

Comment:
do you want a `fontWeight: 'inherit'` here so the comma matches the parent text too?